packages/appkit-utils/src/ErrorUtil.ts

```diff
@@ -0,0 +1,10 @@
+export const ErrorUtil = {
+  UniversalProviderErrors: {
+    UNAUTHORIZED_DOMAIN_NOT_ALLOWED: 'Unauthorized: origin not allowed'
+  },
+  ALERT_ERRORS: {
+    INVALID_APP_CONFIGURATION: 'Invalid App Configuration',
+    ORIGIN_NOT_WHITELISTED: `Origin {{origin}} not found on Allowlist - update configuration`,
+    ORIGIN_NOT_WHITELISTED_SOCIALS: `Origin {{origin}} not found on Allowlist - update configuration to enable social login`
+  }
+}
```

Comment:
is there a better way to handle the replace of origin or other params? 
Right now you have a param here and then you replace it in the code. Thats easy to miss and overlook by someone else. 

What if we change the errors to be functions instead that accept the required params and make it type safe?

Open to another opinion here.

Reply:
Good idea. Updated it to use function instead of string 🙏